core-services/release-controller/_releases/release-ocp-4.15-multi.json

```diff
@@ -49,6 +49,12 @@
       "prowJob": {
         "name": "periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-serial-aws-ovn-heterogeneous"
       }
+    },
+    "multi-azure-ovn-upgrade": {
+      "disabled": true,
```

Comment:
The release-controller will completely ignore jobs that have "disabled": true. Why bother even adding the job?

Reply:
fixed